pacsman/utils.py

```diff
@@ -52,3 +52,19 @@ def _pad_pixel_array_to_square(arr, pad_value=255):
     else:
         padding = ((0, b-a), (0, 0))
     return numpy.pad(arr, padding, mode='constant', constant_values=pad_value)
+
```

Comment:
The strict flake8 on Travis is complaining about a few things here: https://travis-ci.org/innolitics/pacsman/builds/420293836?utm_source=github_status&utm_medium=notification

When I run Tox locally it seems to trip up on pydicom's tests, but I know that if it's gotten to that point it's passed all the flake8 checks. 
```
./pacsman/utils.py:56:1: E302 expected 2 blank lines, found 1
./pacsman/utils.py:61:1: E302 expected 2 blank lines, found 1
./pacsman/utils.py:65:1: E302 expected 2 blank lines, found 1
```